### PR TITLE
Fix #603 refactor play to lessons

### LINKED ISSUE #603: Play should be updated to Lessons
As per new mocks, Topic Play Tab has been updated to Topic Lessons Tab,

Screenshot 2020-01-03 at 12 11 29 PM

So, all files related to that needs to be updated like package name, file names, test cases names, xml files names, variable names, strings.xml, drawables, etc.

In short refactor `Play` with `Lessons`.

### PR DESCRIPTION
**Pull request by PrarabdhGarg**
Fixes #603 

## Explanation
This pull request completes the refactoring process, refactoring files that were left in the previous pull request.

## Checklist

- [x ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x ] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x ] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [ x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [ x] The PR is made from a branch that's **not** called "develop".
- [x ] The PR is made from a branch that is up-to-date with "develop".
- [ x] The PR's branch is based on "develop" and not on any other branch.
- [ x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by rt4914** [comment]
@PrarabdhGarg Why was this PR created, as #609 is similar to this one, any specific reason? Also, please close one of these PR's with proper comment of why you are closing it.

**Comment by PrarabdhGarg** [comment]
@rt4914 is it possible to update the previous pull request with the latest changes I have made?

**Comment by rt4914** [comment]
> @rt4914 is it possible to update the previous pull request with the latest changes I have made?

Yes it is possible, you just need to make a new commit to that same branch and update it.
For example, your branch name is `my-sample-branch`. So you do following things,

`git checkout my-sample-branch`
Now you are in your branch, so make all the changes that you need. After all the changes are finished.

`git add .`
`git commit -m "appropriate commit message"`
`git push origin my-sample-branch`

**Comment by PrarabdhGarg** [comment]
@rt4914 I was not aware of this. Will follow this going forward. 
In this case, I had to create a new pull request as you had suggested changing the branch name

**Comment by PrarabdhGarg** [comment]
I will close the previous Pull request

**Comment by rt4914** [comment]
> @rt4914 I was not aware of this. Will follow this going forward.
> In this case, I had to create a new pull request as you had suggested changing the branch name

Have you taken a look at this document: https://github.com/oppia/oppia-android/wiki#instructions-for-making-a-code-change

This might help.

**Comment by PrarabdhGarg** [comment]
> > @rt4914 I was not aware of this. Will follow this going forward.
> > In this case, I had to create a new pull request as you had suggested changing the branch name
> 
> Have you taken a look at this document: https://github.com/oppia/oppia-android/wiki#instructions-for-making-a-code-change
> 
> This might help.

I will surely read this. Thanks a lot

**Comment by PrarabdhGarg** [comment]
@rt4914 I have closed the previous pull request

**Comment by rt4914** [comment]
> @rt4914 I have closed the previous pull request

@PrarabdhGarg Thanks.

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -157,4 +162,4 @@
       </indentOptions>
     </codeStyleSettings>
   </code_scheme>
-</component>
+</component>
```
Comment (rt4914) [inline .idea/codeStyles/Project.xml]: Revert this file. We do not want these changes.

File: app/src/main/java/org/oppia/app/topic/lessons/TopicLessonsFragment.kt [hunk 2]
```diff
@@ -12,8 +12,8 @@ import javax.inject.Inject
 private const val KEY_CURRENT_EXPANDED_LIST_INDEX = "CURRENT_EXPANDED_LIST_INDEX"
 
 /** Fragment that contains subtopic list for play mode. */
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/topic/lessons/TopicLessonsFragment.kt]: Update this comment to `/** Fragment that contains subtopic list for lessons mode. */`

**Review by rt4914 — CHANGES_REQUESTED** [review]
Looks good to me, some final changes are needed.

1. -> Address all the comments and fix those issues.
2. -> change `play_chapter_view` to `lessons_chapter_view`
3. -> Second step change will need to update the code in `ChapterSummaryAdapter.kt`

**Comment by PrarabdhGarg** [comment]
@rt4914 I have made the required changes

File: .idea/vcs.xml [hunk 3]
```diff
@@ -2,5 +2,6 @@
 <project version="4">
   <component name="VcsDirectoryMappings">
     <mapping directory="" vcs="Git" />
+    <mapping directory="$PROJECT_DIR$/oppia-android" vcs="Git" />
   </component>
-</project>
+</project>
```
Comment (rt4914) [inline .idea/vcs.xml]: Please revert this file changes. We do not want these unnecessary changes.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@PrarabdhGarg `.idea/codeStyles/Project.xml` was not meant to be be deleted. In my earlier comment, by revert I meant that the changes should be reverted and finally the file should be same as before on `develop` branch.

Also revert `.idea/vcs.xml` file changes too, we do not need those changes.

File: app/src/main/java/org/oppia/app/topic/lessons/TopicLessonsFragment.kt (hunk 2 above)
Comment (rt4914) [reply app/src/main/java/org/oppia/app/topic/lessons/TopicLessonsFragment.kt]: @praradbhgarg always make sure that you reply to these comments, some of the comment-replies can be like, what changes did you make, if you have any questions on this comment feel free to mention that, or simply write "DONE" if you had done the suggested changes.

Comment (PrarabdhGarg) [reply app/src/main/java/org/oppia/app/topic/lessons/TopicLessonsFragment.kt]: Ok. Will take care next time

Comment (PrarabdhGarg) [reply app/src/main/java/org/oppia/app/topic/lessons/TopicLessonsFragment.kt]: Done

File: .idea/vcs.xml (hunk 3 above)
Comment (PrarabdhGarg) [reply .idea/vcs.xml]: I will do this. I am sorry for the confusion

File: app/src/main/java/org/oppia/app/topic/lessons/TopicLessonsFragment.kt (hunk 2 above)
Comment (rt4914) [reply app/src/main/java/org/oppia/app/topic/lessons/TopicLessonsFragment.kt]: No issues, we already knew that this PR is easy, but the focus behind such PRs to make you aware of how oppia as a product is and how we normally communicate on PRs. It's completely fine.

File: .idea/vcs.xml (hunk 3 above)
Comment (rt4914) [reply .idea/vcs.xml]: No issues. To make PRs more effective, before sending the PR for review its nice to go to the pull request, click on "Files Changed" and try to self review it first and whatever mistakes you find, fix them and then send it for review. This will help you a lot in becoming a good developer.

**Comment by PrarabdhGarg** [comment]
@rt4914 I have added the files again and made the small changes

**Comment by PrarabdhGarg** [comment]
Are the merge conflicts coming because I didn't update my branch with the latest develop branch? If yes, should I be solving them, or you would be solving them while merging the pull request?

**Comment by rt4914** [comment]
> Are the merge conflicts coming because I didn't update my branch with the latest develop branch? If yes, should I be solving them, or you would be solving them while merging the pull request?

It would be nice if we can solve it right now only.

File: .idea/codeStyles/Project.xml (hunk 1 above)
Comment (rt4914) [reply .idea/codeStyles/Project.xml]: @PrarabdhGarg This file is still here. We do not want to make any changes to this file. So please revert back this file to its original code.

File: .idea/vcs.xml (hunk 3 above)
Comment (rt4914) [reply .idea/vcs.xml]: Same here. This file changes are also not needed.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@PrarabdhGarg PTAL

**Comment by PrarabdhGarg** [comment]
> @PrarabdhGarg PTAL

@rt4914 Since I had deleted the files in my previous commit, I just copied the files from the current develop branch, and committed them. I guess now they are the same as that on the develop branch

**Comment by PrarabdhGarg** [comment]
@rt4914 I have also resolved the merge conflicts

**Comment by rt4914** [comment]
> > @PrarabdhGarg PTAL
> 
> @rt4914 Since I had deleted the files in my previous commit, I just copied the files from the current develop branch, and committed them. I guess now they are the same as that on the develop branch

You can go to `Files changed` tab in this PR and check for following files:
`.idea/codeStyles/Project.xml `
`.idea/vcs.xml `

These files are getting changed. We do not want any change to these files.

**Comment by rt4914** [comment]
> @rt4914 I have also resolved the merge conflicts

@PrarabdhGarg thanks.

**Comment by PrarabdhGarg** [comment]
@rt4914 I have reverted the files. How do I ensure that any local changes in these files are not pushed in the future? Should I add them to the git ignore file?

**Comment by rt4914** [comment]
> @rt4914 I have reverted the files. How do I ensure that any local changes in these files are not pushed in the future? Should I add them to the git ignore file?

Thanks a lot. Actually these files should not be added to git ignore. Also, mostly it should get fixed in your next PR. If it does not get fixed please inform us and we will try to come up with some solution.

**Review by rt4914 — APPROVED** [review]
LGTM. Thanks @PrarabdhGarg
